### ISSUE TITLE
Not convinced it will solve the employer problem.

### ISSUE THREAD

**Issue by tiagov8** [wicksipedia/PointBank#7]
I can see the employee problem solved - more incentive to do the tasks.
But not the employer - the time to build/maintain this app will be significant compared to the benefit.

**Comment by StevenA89** [comment]
This in my opinion is a waste of time. I do not see it adding value.

**Comment by wicksipedia** [comment]
@tiagov8  👍 it does look like extra admin to make it worthwhile
I feel like there isn't enough benefit to make me want to use it

**Comment by isonaj** [comment]
Seems like a lot of overhead to produce a data stream that already exists.

**Comment by isonaj** [comment]
I'm also not sure how this app would help the employer know whether to prioritise standup desks or fingerprint scanners for the door.

**Comment by camillars** [comment]
I felt like it's going to take a lot of time to build this app comparing it to the benefit. +1 with all above.

**Comment by kikibianc** [comment]
I have seen systems like this used on other companies (much bigger companies), and the results were quite good.
However, the rewards were much better as well - accumulating points gave you many more benefits:
- More leave time;
- Free meals;
- If you are on the top of the leaderboards for x amount of time, you win a trip to somewhere;
- Even more money at the end of the month - I saw a particular company that you could convert the points to money after some time.

The idea is good, but it needs to be well implemented and have worthy rewards.

**Comment by LiamElliott-au** [comment]
I see this as a platform that, yes, will incur extra overhead to get up and running, but is also very scale able.

**Comment by PatrickZhao1989** [comment]
Good side of this application is that it could potentially save a lot of admin's time to create, distribute and chase for the questionnaires.  It also in some degree encourages employee to complete some task. 
Downside is that it still does not have a solution when an employee just does not have time to complete the task even the task has good rewards in which case the **employer** has to chase.

**Comment by hoangphan84** [comment]
I agree that this will add a lot of overheads to setup and maintain to the employer, that will make them less willing to invest in this product.

**Comment by calumjs** [comment]
@tiagov8 (emphasis mine) 
> I can see the employee problem solved - more incentive to do the tasks.
> But not the **employer** - the time to build/maintain this app will be significant compared to the benefit.

Keep in mind when evaluating the time taken that we should separate SSW as developers and SSW as users...an app that takes a lot of dev time but little admin time for users could still be beneficial to the users (but maybe still not worthwhile for us to invest the dev time in creating it).

@kikibianc 
> I have seen systems like this used on other companies (much bigger companies), and the results were quite good.
> However, the rewards were much better as well - accumulating points gave you many more benefits:
> 
> * More leave time;
> * Free meals;
> * If you are on the top of the leaderboards for x amount of time, you win a trip to somewhere;
> * Even more money at the end of the month - I saw a particular company that you could convert the points to money after some time.
> 
> The idea is good, but it needs to be well implemented and have worthy rewards.

See #6

**Comment by christoment** [comment]
I also agree that although this could add some value for us, I'm not sure that we will have a good return compared to the time invested (dev time wise).

I can see the value for the company such as SSW where we have a weekly task, but I have a hard time seeing the business opportunity for this kind of app, as not many companies have a regular questionnaire (correct me if I'm wrong though).
